WordPress/src/test/java/org/wordpress/android/ui/reader/repository/usecases/GetDiscoverCardsUseCaseTest.kt

```diff
@@ -56,6 +60,11 @@ class GetDiscoverCardsUseCaseTest {
                 .thenReturn(ReaderConstants.JSON_CARD_POST)
         whenever(mockedInterestsCardJson.getString(ReaderConstants.JSON_CARD_TYPE))
                 .thenReturn(ReaderConstants.JSON_CARD_INTERESTS_YOU_MAY_LIKE)
+
```

Comment:
This was put here so that the tests would pass in this PR. Since this use case won't need the `ReaderTagRepository` when the API endpoint for the interests you may like is working, I will remove this logic at the time of its implementation.